__tests__/app.test.js

```diff
@@ -84,3 +84,30 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  test(`200: Responds with an array of objects, each object should have the following property: username, name avatar_url`, () => {
```

Comment:
Maybe even here? responds with an array of USER objects would be more descriptive.